catalogue/webapp/services/iiif/transformers/manifest.ts

```diff
@@ -65,14 +64,14 @@ export function transformManifest(
   const structures = manifestV2?.structures || [];
 
   // V3
+  const title = manifestV3?.label ? getTitle(manifestV3.label) : '';
```

Comment:
nit: I wonder if it would be worth pushing this ternary down into `getTitle`, so here you'd have:

```typescript
const title = getTitle(manifestV3?.label);
```

and the function absorbs the `if (isUndefined) return ''` logic

Reply:
I had it like that originally and reverted to this. Couldn't decide where it was better to have the possibly undefined-ness. Ultimately I thought it was nicer to have a required argument for the function, but couldn't really articulate why. Is there a good rule of thumb/best practice?